quickstart.org

```diff
@@ -55,57 +82,97 @@ To buy:
    + Mozzarella
 #+END_SRC
 
-Lists can also contain checkboxes: ~[ ]~, ~[-]~, ~[X]~
+Lists can contain checkboxes ~[ ]~, ~[-]~, ~[X]~.
 #+BEGIN_SRC org
 - [ ] do
 - [-] the
 - [X] thing
 #+END_SRC
-* Images
-Just link to an image, and don't give a description. Seriously, that's it. \\
-~[[https://upload.wikimedia.org/wikipedia/commons/5/5d/Konigsberg_bridges.png]]~
 
-#+attr_html: :height 120px
-[[https://upload.wikimedia.org/wikipedia/commons/5/5d/Konigsberg_bridges.png]]
+Lists can contain tags (and checkboxes at the same time).
+#+BEGIN_SRC org
+- [ ] fruits :: get apples
+- [X] veggies :: get carrots
+#+END_SRC
+
 * Links
-Just put a target in between two square brackets, like so: ~[[target]]~. Want to
-add a description? Just add a ~[description]~ after the target, i.e.
-~[[target][desc]]~.
+To create a link put the target between two square brackets, like so: ~[[target]]~.
+To add a description add it in square brackets after the target ~[[target][desc]]~.
+
+The order is easy to remember because it is the same as an html =<a>=
+tag ~<a href="target">desc</a>~ and you can't mix up brackets and
+parentheses because there are only square brackets!
 
-Org can handle a variety of different types of targets, which you use by giving
-~[target]~ as ~[type:target]~. When no ~type:~ is provided, org just searches for a
-heading which matches ~target~.
+Org supports a wide variety of types of link targets, and you can also
+[[https://orgmode.org/manual/Adding-Hyperlink-Types.html][add your own]].
+Link types are indicated with a ~type:~ prefix, e.g. ~[type:target]~.
```

Comment:
Minor niggle: This should be `i.e.` not `e.g.`.

Reply:
`i.e.,` iirc but yes, since there isn't any other way to do it so it is not an example.